polkadot-parachains/encointer-runtime/src/lib.rs

```diff
@@ -181,7 +258,7 @@ parameter_types! {
 pub struct BaseFilter;
 impl Contains<Call> for BaseFilter {
 	fn contains(_c: &Call) -> bool {
-		true
+		matches!(_c, Call::Proxy(_))
```

Comment:
This will disable all extrinsics except for proxy calls. We don't want that. Just leave it as true for now, to allow everything. :)